controller/users_blackbox_test.go

```diff
@@ -51,6 +54,48 @@ func (s *TestUsersSuite) SecuredServiceAccountController(identity account.Identi
 	return svc, NewUsersController(svc, s.GormDB, s.Configuration)
 }
 
+func (s *TestUsersSuite) TestDeleteUsersBadRequest() {
+	// given
+	fxt := tf.NewTestFixture(s.T(), s.DB, tf.Users(1), tf.Identities(1))
+	secureService, secureController := s.SecuredServiceAccountController(*fxt.Identities[0])
+	// when
+	emptyUsername := ""
+	test.DeleteUsersBadRequest(s.T(), secureService.Context, secureService, secureController, emptyUsername)
+}
+
+func (s *TestUsersSuite) TestDeleteUsersOK() {
+	// given
+	fxt := tf.NewTestFixture(s.T(), s.DB, tf.Users(1), tf.Identities(1), tf.Spaces(1))
+	// when
+	secureService, secureController := s.SecuredServiceAccountController(*fxt.Identities[0])
+	test.DeleteUsersOK(s.T(), secureService.Context, secureService, secureController, fxt.Identities[0].Username)
+	// then
+	_, err := s.userRepo.Load(context.Background(), fxt.Users[0].ID)
```

Comment:
Please also check that `CheckExists` returns an error on the user, identity and space repo. 

```go
_, err := s.userRepo.CheckExists(context.Background(), fxt.Users[0].ID)
require.Error(s.T(), err, "User should not exist")
// ...
```

Reply:
fixed 1f6363d2d1dcf01302348cd164ed112838f7eede